src/buildingriskassessment/hooks/CalendarHooks.js

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import { BASE_URL } from '../../config/APIConfig';
+import { useAPI } from '../../common/hooks/API';
+import { convertUTCDateToLocalDate } from '../../utils/Time.js';
+
+export const useCalendarHooks = () => {
+    const { loadData } = useAPI();
+    const [
+        riskAssessmentScheduleList,
+        setRiskAssessmentScheduleList,
+    ] = useState([]);
+
+    const [agendaItems, setAgendaItems] = useState([]);
+
+    async function getAssociatesRiskAssessmentSchedules(id) {
+        const riskAssessmentSchedules = await loadData(
+            `${BASE_URL}/getRiskAssessmentSchedulesOfSiteMaintenanceAssociatesOfManager?id=${id}&activeSchedules=true`,
+            'GET'
+        );
+        return riskAssessmentSchedules;
+    }
+
+    function formatRiskAssessmentSchedules(riskAssessmentSchedules) {
```

Comment:
This function already exists inside of `src/riskassessmentcalendar/hooks/CalendarHooks.js`, so no need to duplicate here